### DISCUSSION TITLE
Is it possible to output a warning message when running a test?

### DISCUSSION THREAD

**Discussion by JanisE** [Codeception/Codeception#6568] [Q&A]
Is it possible to output a warning message when running a test? Something like `InitTemplate::sayWarning`.

I'm trying to use `$I->comment()`, but it gets output only in the case there are test errors. If all the test checks are successful, then the comment does not appear at all.

**Comment by Naktibalda** [comment]
You could use `Codeception\Lib\Notification::warning(string $message, string $location)` method.

The message is displayed after the last test in test suite, like this:
![image](https://user-images.githubusercontent.com/395992/188286768-247dd589-883d-40e1-a61c-479a4fd2ec4b.png)
